File: projects/m1/004-units-of-time/solutions/stefandifede/units-of-time.js
const days = prompt("insert number of days");
const hours = prompt("insert number of hours");
const minutes = prompt("insert number of minutes");
const seconds = prompt("insert number of seconds");

const oneMinute = 60;
const oneHour = 60 * oneMinute;
const oneDay = 24 * oneHour; 

function calculateAmountOfSeconds() {
    if (isNaN(days) || isNaN(hours) || isNaN(minutes) || isNaN(seconds)) {
        alert("insert a value of days, hours, minutes and seconds")
    } else {alert("Total of seconds " + ((days * oneDay) + (hours * oneHour) + (minutes * oneMinute) + seconds))}
}

calculateAmountOfSeconds()

File: projects/m1/004-units-of-time/solution/NoelLapedota/units-of-time.py
days_user = int(input('Insert a number of days: '))
hours_user = int(input('Insert a number of hours: '))
minutes_user = int(input('Insert a number of minutes: '))
second_user =int(input('insert a number of seconds'))
total_seconds=(days_user * days_user) + (hours_user * 3600) + (minutes_user * 60) + second_user
print(f'Total seconds are : {total_seconds}')